tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/statements/assign/AssignStmtTest.java

```diff
@@ -270,6 +270,11 @@ public void assignAnyToUnionWithErrorAndAny() {
         BRunUtil.invoke(result, "assignAnyToUnionWithErrorAndAny");
     }
 
+    @Test()
+    public void assignVarInQueryExpression() {
```

Comment:
Shall we change to 
```suggestion
    public void testAssignQueryExpressionToVar() {
```

Reply:
Will update.

Reply:
Added. da824d8c601d69e9fa8df16a5167e6e999de48c3